internal/engine/upper.go

```diff
@@ -935,6 +935,7 @@ func handleDockerComposeLogAction(state *store.EngineState, action DockerCompose
 
 	dcState, _ := ms.ResourceState.(dockercompose.State)
 	ms.ResourceState = dcState.WithCurrentLog(model.AppendLog(dcState.CurrentLog, action, state.LogTimestamps))
+	state.Log = model.AppendLog(state.Log, action, state.LogTimestamps)
```

Comment:
i'm pretty sure this logs to state.Log twice, because it already gets logged to the global log writer here:
https://github.com/windmilleng/tilt/blob/master/internal/engine/docker_compose_log_manager.go#L125